app/src/main/java/com/framgia/music_49/data/source/remote/FetchDataRemoteDataSource.java

```diff
@@ -13,15 +13,19 @@
 import org.json.JSONArray;
 import org.json.JSONObject;
 
-import static com.framgia.music_49.Utils.Constant.CLIENT_ID;
+import static com.framgia.music_49.utils.Constant.CLIENT_ID;
 
-public class FetchDataRemote extends AsyncTask<String, Void, List<Song>>
-        implements SongRepository.SongRemote {
+public class FetchDataRemoteDataSource extends AsyncTask<String, Void, List<Song>>
```

Comment:
rename lại tên class thể hiện chức năng của nó e
```suggestion
public class GetJsonFromUrl extends AsyncTask<String, Void, List<Song>>
```